setup.py

```diff
@@ -57,7 +57,8 @@ def create_version_file():
     'pandas>=0.24.0,<1.0',
     'psutil>=5.0.0',
     'scikit-learn>=0.22.0,<0.23',
-    'networkx>=2.3,<3.0'
+    'networkx>=2.3,<3.0',
+    'faiss-cpu>=1.6.3'
```

Comment:
May decide to remove this as a required dependency, but can keep for the moment. Will have to identify how heavy the package is.